docs/zh/UserGuide/IoTDB-SQL-Language/DML-Data-Manipulation-Language.md

```diff
@@ -443,8 +480,10 @@ It costs 0.014s
 | 函数名 | 必要的属性参数                                               | 输出序列类型               | 功能描述                                                     |
 | ------ | ------------------------------------------------------------ | -------------------------- | ------------------------------------------------------------ |
 | CONST  | `value`: 输出的数据点的值 <br />`type`: 输出的数据点的类型，只能是 INT32 / INT64 / FLOAT / DOUBLE / BOOLEAN / TEXT | 由输入属性参数 `type` 决定 | 根据输入属性 `value` 和 `type` 输出用户指定的常序列。        |
-| PI     | 无                                                           | DOUBLE                     | 常序列的值：`π` 的 `double` 值，圆的周长与其直径的比值，即圆周率，等于 *Java标准库* 中的`Math.PI`。 |
-| E      | 无                                                           | DOUBLE                     | 常序列的值：`e` 的 `double` 值，自然对数的底，它等于 *Java 标准库*  中的 `Math.E`。 |
+| PI     | 无                                                           | DOUBLE                     | 常序列的值：`π` 的 `double` 值，圆的周长与其直径的比值，即圆周率，等于 *
+Java标准库* 中的`Math.PI`。 |
```

Comment:
The table format is changed unexpectedly.

Please check this and other places :D

Reply:
This is the fault of IDEA markdown auto-check. I have rewritten the docs.